t/01_run.t

```diff
@@ -578,4 +574,18 @@ subtest 'process in process' => sub {
     is ($p->read_all_stdout(), 'DONE', "Use ReadWriteProcess inside of ReadWriteProcess(code=>'')");
 };
 
+subtest 'execute exeption handling' => sub {
+    throws_ok {
+        process(execute => '/I/do/not/exists')->start()->wait_stop()->exit_status();
+    } qr/No such file or directory/, 'Execute throw exeption, if executable does not exists';
```

Comment:
```suggestion
    } qr/No such file or directory/, 'Execute throw exception, if executable does not exists';
```

It also looks like the error message isn't exactly like this in all Perl versions, e.g.  the CI run for Perl 5.22 fails with:

```
    #   Failed test 'Execute throw exeption, if executable does not exists'
Error:     #   at t/01_run.t line 580.
    # expecting: Regexp ((?^:No such file or directory))
Error:     # found: open3: exec of /I/do/not/exists failed at lib/Mojo/IOLoop/ReadWriteProcess.pm line 127.
```

Reply:
thx, done